app/views/main.scala.html

```diff
@@ -24,14 +24,12 @@
         <div class="govuk-header__container govuk-width-container">
 
             <div class="govuk-header__tna-logo">
-                <a href="/" class="govuk-header__link">
-                    <span>
-                        <image src="/assets/images/tna-horizontal-white-logo.svg" class="govuk-header__tna-logo-image"></image>
-                    </span>
+                <a href="/" class="govuk-header__link govuk-header__link--homepage">
```

Comment:
This link has an underline when you hover over it, which looks a bit odd because it's an image rather than text:

<img width="232" alt="Screenshot 2020-06-10 at 11 33 34" src="https://user-images.githubusercontent.com/754712/84258121-53150600-ab0e-11ea-996b-19d9dfbb971a.png">

Reply:
This was a case of the default govuk scss files including an underline on the logo + text, so I had to override it, which is fine as it is fixed now.